### ISSUE TITLE
Align Design with New Clubs Site

### ISSUE THREAD

**Issue by brianking** [mozilla/activate.mozilla.community#58]
My understanding is that the new Clubs site is being built on the same technology as the Activate site. I think it would make sense for them to have a similar design.

Can we share CSS, image assets, and perhaps even code or markup?

Bring in @comzeradd for initial comments on feasibility.

cc @LucyeoH and @bacharakis fyi.

**Comment by comzeradd** [comment]
I started "playing" around with the site. For start I introduced more or less the same typographic rules we have at the Campus Clubs website and at Mozilla in general.

I also changed the color theme to match the work the Open Design does to create an identity for the Activate campaign. Maybe we could ask them to create a top header if we want to align more with the Campus Clubs layout.

![home](https://cloud.githubusercontent.com/assets/939357/18511116/a5d619dc-7a8a-11e6-8fd3-aeb5cd3aadb7.png)
